package.json

```
@@ -62,7 +62,7 @@
     "@harness/monaco-yaml": "^1.0.0",
     "@harness/ng-tooltip": "^1.31.62",
     "@harness/telemetry": "^1.0.42",
-    "@harness/uicore": "^3.98.1",
+    "@harness/uicore": "^3.99.2",
```

Why is this bumping the minor and patch? It looks like this is a new feature, so it should bump the minor version
```suggestion
    "@harness/uicore": "^3.99.0",
```